README.markdown

```diff
@@ -21,19 +22,20 @@ Contents
 *   `README.markdown`: these post-modernist existential rants
 *   `LICENSE`: a bedtime story written to give your lawyer sweet dreams
 *   `doc/Befunge-93.markdown`: the specification for Befunge-93, such as it is
-*   `src/bef.c`: Befunge-93 reference interpreter/debugger v2.23 source code
-*   `src/bef2c.c`: Befunge-93 to ANSI C compiler v0.94 source code
-*   `src/befprof.c`: Befunge-93 profiler v0.94 source code
+*   `src/bef.c`: Befunge-93 reference interpreter/debugger v2.24 source code
+*   `src/bef2c.c`: Befunge-93 to ANSI C compiler v1.0 source code
+*   `src/befprof.c`: Befunge-93 profiler v1.0 source code
 *   `eg/*`: Various and sundry contributed Befunge-93 programs
 
 For More Information
 --------------------
 
-See Cat's Eye Technologies' official [Befunge-93 Project Page][].
+See the official entry for [Befunge-93][] at [Cat's Eye Technologies][].
```

Comment:
I'm not sure if this is too late to fix, but this link text change to _Befunge-93_ has caused the link to break.